keyboards/keychron/k6/keymaps/ansi/keymap.c

```diff
@@ -23,58 +23,65 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
-                  {   KC_CAPS,   KC_A,      KC_S,      KC_D,    KC_F,     KC_G,    KC_H,     KC_J,    KC_K,    KC_L,      KC_SCLN,   KC_QUOT,   KC_NO,     KC_ENT,    KC_NO,     KC_PGUP   },
-                  {   KC_LSFT,   KC_NO,     KC_Z,      KC_X,    KC_C,     KC_V,    KC_B,     KC_N,    KC_M,    KC_COMM,   KC_DOT,    KC_SLSH,   KC_NO,     KC_RSFT,   KC_UP,     KC_PGDOWN },
-                  {   KC_LCTL,   KC_LALT,   KC_LGUI,   KC_NO,   KC_NO,    KC_NO,   KC_SPC,   KC_NO,   KC_NO,   KC_NO,     KC_RGUI,   MO(_FNM),  MO(_FN2),  KC_LEFT,   KC_DOWN,   KC_RGHT   }
+       [_MAC] = { {   _______,   _______,   _______,   _______,   _______,    _______,   _______,   _______,   _______,   _______,   _______,   _______,    _______,   _______,    XXXXXXX,    _______   },
+                  {   _______,   _______,   _______,   _______,   _______,    _______,   _______,   _______,   _______,   _______,   _______,   _______,    _______,   _______,    XXXXXXX,    _______   },
+                  {   _______,   _______,   _______,   _______,   _______,    _______,   _______,   _______,   _______,   _______,   _______,   _______,    XXXXXXX,   _______,    XXXXXXX,    _______   },
+                  {   _______,   XXXXXXX,   _______,   _______,   _______,    _______,   _______,   _______,   _______,   _______,   _______,   _______,    XXXXXXX,   _______,    _______,    _______   },
+                  {   _______,   KC_LALT,   KC_LGUI,   XXXXXXX,   XXXXXXX,    XXXXXXX,   _______,   XXXXXXX,   XXXXXXX,   XXXXXXX,   _______,   MO(_FNM),   _______,   _______,    _______,    _______   }
                 },
-       [_FNM] = { {   KC_GESC,   KC_SLCK,   KC_PAUS,   KC_APP,  KC_NO,    RGB_VAD, RGB_VAI,  KC_MPRV, KC_MPLY, KC_MNXT,   KC_MUTE,   KC_VOLD,   KC_VOLU,   KC_NO,     KC_NO,     RGB_TOG   },
-                  {   KC_NO,     KC_NO,     KC_NO,     KC_NO,   KC_NO,    KC_NO,   KC_NO,    KC_NO,   KC_NO,   KC_NO,     KC_INS,    KC_DEL,    KC_END,    KC_NO,     KC_NO,     KC_NO     },
-                  {   KC_NO,     KC_NO,     KC_NO,     KC_NO,   KC_NO,    KC_NO,   KC_NO,    KC_NO,   KC_NO,   KC_NO,     KC_NO,     KC_NO,     KC_NO,     KC_NO,     KC_NO,     KC_NO     },
-                  {   KC_TRNS,   KC_NO,     KC_NO,     KC_NO,   KC_NO,    KC_NO,   KC_NO,    KC_NO,   KC_NO,   KC_NO,     KC_NO,     KC_NO,     KC_NO,     KC_NO,     RGB_VAI,   KC_NO     },
-                  {   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_NO,   KC_NO,    KC_NO,   KC_NO,    KC_NO,   KC_NO,   KC_NO,     KC_RCTRL,  KC_NO,     KC_NO,     RGB_HUD,   RGB_VAD,   RGB_HUI   }
+       [_FNW] = { {   KC_GRV,    KC_BRIU,   KC_BRID,   KC_TASK,   KC_FLXP,    RGB_VAD,   RGB_VAI,   KC_MPRV,   KC_MPLY,   KC_MNXT,   KC_MUTE,   KC_VOLD,    KC_VOLU,   XXXXXXX,    XXXXXXX,    RGB_TOG   },
+                  {   XXXXXXX,   XXXXXXX,   XXXXXXX,   XXXXXXX,   XXXXXXX,    XXXXXXX,   XXXXXXX,   XXXXXXX,   XXXXXXX,   XXXXXXX,   KC_INS,    KC_DEL,     KC_END,    XXXXXXX,    XXXXXXX,    XXXXXXX   },
```

Comment:
no need for `KC_NO,` or `XXXXXXX,`. Just leave the spot blank. Much better readability